specification/vmware/resource-manager/Microsoft.AVS/stable/2020-03-20/vmware.json

```diff
@@ -1807,8 +1965,7 @@
         "nsxtPassword": {
           "description": "NSX-T Manager password",
           "type": "string",
-          "readOnly": true,
-          "x-ms-secret": true
```

Comment:
I need to roll this change back according to @pilor in https://github.com/Azure/azure-rest-api-specs/pull/10866#discussion_r507966702.

Reply:
But CI fails with it in place, so I'm not sure what to do @ruowan.

Reply:
I put these back and suppressed the errors as recommended.